stable/redis-ha/templates/redis-master-appliance-vpc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-master-appliance-vpc
+  annotations:
+    cloud.google.com/load-balancer-type: "Internal"
+spec:
+  ports:
+  - port: 6379
+    protocol: TCP
+    targetPort: 6379
+  selector:
+    app: "redis-ha"
+    redis-node: "true"
+    redis-role: "master"
+    release: "{{ .Release.Name }}"
+  type: LoadBalancer
```

Comment:
Make this a ClusterIP by default to prevent unwanted exposure OOTB.

Reply:
Done. Added appliance.vpcMode flag to control this. Defaults to ClusterIP.